docs/source/querying/querybuilder/queryhelp.rst

```diff
@@ -176,14 +162,17 @@ What do you have to specify:
     but only the ones that were added
     after a certain time `t` and have an id higher than 50::
 
+        from aiida.common import timezone
```

Comment:
When you update this, you should update the text above as well, which mentions the (now deleted) time `t`.

Reply:
right, thanks!